cloudbot/util/timeparse.py

```diff
@@ -159,16 +159,19 @@ def time_parse(string, granularity='seconds'):
             if all(v.isdigit() for v in list(mdict.values()) if v):
                 return sign * sum([MULTIPLIERS[k] * int(v, 10) for (k, v) in
                                    list(mdict.items()) if v is not None])
+
             # if SECS is an integer number
-            elif ('secs' not in mdict or
-                  mdict['secs'] is None or
-                  mdict['secs'].isdigit()):
+            if (
```

Comment:
Why does this if need to be on multiple lines?

Reply:
what exactly are you highlighting here

Reply:
The multiline if. Github hates me. ie, 
```suggestion
            if  'secs' not in mdict or mdict['secs'] is None or  mdict['secs'].isdigit():
```

Reply:
because as I said in other comments, I'm limiting the changes to what I set out to do, specifically just removing the `return-else` statements